src/service/api_client/site_api_service.py

```diff
@@ -0,0 +1,56 @@
+from http import HTTPStatus
+
+import httpx
+
+from service.api_client.base import AbstractAPIService
+from service.api_client.models import BillStat, MonthStat, UserData, UserMonthStat, UserWeekStat, WeekStat
+
+
+class SiteAPIService(AbstractAPIService):
+    async def get_bill(self) -> BillStat:
+        url = f"{self.site_url}/tgbot/bill"
+        async with httpx.AsyncClient() as client:
+            response = await client.get(url)
+            response = await response.json()
+            return WeekStat(response["telegram_name"])
```

Comment:
тебе же надо вернуть BillStat, надо из так 
```suggestion
            return BillStat(telegram_name=response["telegram_name"])
```